docs/migration-guides/qiskit-2.0-features.mdx

```diff
@@ -0,0 +1,122 @@
+            (1,): InstructionProperties(error=.00002, duration=6e-8)
+        }
+    )
+
+    pm = generate_preset_pass_manager(target=target)
+    ```
+
+  b. Use [`GenericBackendV2`](/api/qiskit/qiskit.providers.fake_provider.GenericBackendV2) to quickly build a 
+     generic target with pre-filled errors and durations. These can also be updated post-construction:
+
+    ```python
+    from qiskit.providers.fake_provider import GenericBackendV2
+    from qiskit.transpiler import generate_preset_pass_manager
+
+    target = GenericBackendV2(num_qubits=2, basis_gates=["x", "cx"]).target
+    target.update_instruction_properties("x", (0,), InstructionProperties(0.00001))
+    target.update_instruction_properties("x", (1,), InstructionProperties(0.00002))
+    target.update_instruction_properties("cx", (0, 1), InstructionProperties(0.0001))
+
+    pm = generate_preset_pass_manager(target=target)
+    ```
+
+2. The `instruction_durations` and `timing_constraints` input arguments are no longer supported. You should use the 
+  `target` or `backend` arguments instead. The alternatives are constructing a custom [`Target`](/api/qiskit/qiskit.transpiler.Target) from
+  scratch, constructing a generic backend, and, in addition to these, it is possible to build a custom target with 
+  `instruction_durations` and/or `timing_constraints` using the `Target.from_configuration` method:
+
+    ```python
+    from qiskit.transpiler import Target, generate_preset_pass_manager
+    from qiskit.transpiler.timing_constraints import TimingConstraints
+    from qiskit.transpiler.instruction_durations import InstructionDurations
+
+    instruction_durations = InstructionDurations([("x", None, 200), ("cx", None, 1000)], dt=1e-7)
+
+    timing_constraints = TimingConstraints(
+            granularity=1, min_length=1, pulse_alignment=1, acquire_alignment=1
+        )
+
+    target = Target.from_configuration(num_qubits=2, 
+                                        basis_gates=["x", "cx"], 
+                                        instruction_durations=instruction_durations, 
+                                        timing_constraints=timing_constraints)
+    
+    pm = generate_preset_pass_manager(target=target)
+    ```
+
+3. Providing `coupling_map` and/or `basis_gates` along with a `backend` is discouraged, and from 
+  v2.0 a `UserWarning` will be raised. In these cases there are multiple sources of truth, 
+  and information such as gate errors and durations from the backend cannot be resolved based on the
+  information given.
+  The suggested alternative is to define a custom target that combines the chosen constraints either
+  from scratch, using [`Target.from_configuration`](/api/qiskit/qiskit.transpiler.Target#from_configuration) or [`GenericBackendV2`](/api/qiskit/qiskit.providers.fake_provider.GenericBackendV2). 
+
+- The `basis_gates` argument no longer accepts custom basis gates. The suggested alternative
```

Comment:
We seem to swap from a numbered list to a bullet one - is that deliberate, or is one meant to be nested?

Reply:
I started with bullets and then decided to use a numbered list, so I probably forgot a couple, let me review that.